docs/source/guides/model-loading-and-management.rst

```diff
@@ -79,6 +79,8 @@ BentoML provides an efficient mechanism for loading AI models to accelerate mode
                  self.model = AutoModelForSequenceClassification.from_pretrained(self.model_ref)
                  self.tokenizer = AutoTokenizer.from_pretrained(self.model_ref)
 
+      By default, ``HuggingFaceModel`` returns the downloaded model path as a string, which means you can directly pass the path into libraries like ``transformers`` for model loading.
```

Comment:
nit: should we rename `model_ref` to `model_path` for HuggingFaceModel, given that it is not really a `ref`?